src/Nancy.Swagger.Annotations/SwaggerObjects/AnnotatedResponse.cs

```diff
@@ -9,7 +9,8 @@ namespace Nancy.Swagger.Annotations.SwaggerObjects
 {
     public class AnnotatedResponse : global::Swagger.ObjectModel.Response
     {
-        public int StatusCode { get; set; }
+        private int StatusCode { get; set; }
+        public int GetStatusCode() => StatusCode;
```

Comment:
Can you explain the purpose of your change? 

I don't know what you mean by: "Responses all had StatusCode: xxx, I removed this"
Can you show the before and after too.